test/e2e/suite_test.go

```diff
@@ -41,24 +41,26 @@ import (
 )
 
 const (
-	kubeconfigEnvVar    = "KUBECONFIG"
-	reportDirEnvVar     = "ARTIFACTS"
-	testMigrationEnvVar = "TEST_MIGRATION"
-	testWindowsEnvVar   = "TEST_WINDOWS"
-	cloudNameEnvVar     = "AZURE_CLOUD_NAME"
-	defaultReportDir    = "/workspace/_artifacts"
-	inTreeStorageClass  = "kubernetes.io/azure-disk"
-	buildV2Driver       = "BUILD_V2"
+	kubeconfigEnvVar        = "KUBECONFIG"
+	reportDirEnvVar         = "ARTIFACTS"
+	testMigrationEnvVar     = "TEST_MIGRATION"
+	testWindowsEnvVar       = "TEST_WINDOWS"
+	cloudNameEnvVar         = "AZURE_CLOUD_NAME"
+	defaultReportDir        = "/workspace/_artifacts"
+	inTreeStorageClass      = "kubernetes.io/azure-disk"
+	buildV2Driver           = "BUILD_V2"
+	useOnlyDefaultScheduler = "USE_ONLY_DEFAULT_SCHEDULER"
```

Comment:
From what I understand, in scheduler extender we are going to try to short-circuit the cases where pod scheduling will not rely on V2 driver components, i.e. there are no requests for volumes with v2 provisioner or azNodes/azVolumes are not available. To verify that the scheduler extender will work as expected for such cases as well, I think it is helpful to run the tests for V1 driver with scheduler extender. This would allow us to check both the functionality and the impact of the scheduler extender on performance.

Reply:
OK. Sounds good. We can leave this as-is.